### DISCUSSION TITLE
11.0 Preview 6 version did not find the method about OpenGL glUniform 1 i

### DISCUSSION THREAD

**Discussion by qqJoker** [AvaloniaUI/Avalonia#10898] [Q&A]
When I tried to use OpenGLES to render `**YUVJ420P**` data, I found that there is no `glUniform1i()` method in the GlInterface class, only a `glUniform1f()` method. I try to use the  **`glUniform1f()`** method, but OpenGL will prompt: `Error(1282)`. Is there any way to replace `glUniform1i()` in the **11.0 Preview 6** version, looking forward to reply, thank you!

**Comment by qqJoker** [comment]
In version 0.10.18, I tried to use Silk.Net to render YUVJ420P, and the video stream can be played normally, but when playing multiple RTSP stream addresses at the same time, the interface will be stuck and cannot be refreshed normally, and it often Occasional anomalous events will occur.

**Comment by maxkatz6** [comment]
https://github.com/AvaloniaUI/Avalonia/discussions/10646#discussioncomment-5290679
